src/components/SubHeader.tsx

```diff
@@ -4,13 +4,8 @@ import { Box, Heading } from 'rendition';
 export const SubHeader = () => {
 	return (
 		<Box width="100%" bg="#333333">
-			<Box mx="auto" pb={3} px={3} width="100%" maxWidth="1215px">
-				<Heading.h1
-					style={{ lineHeight: '1.28' }}
-					color="#fff"
-					fontSize={[5, 6, 7, 7]}
-					bold
-				>
+			<Box mx="auto" pb={3} px={3} width="100%" maxWidth="1280px">
+				<Heading.h1 color="#fff" fontSize={[4, 5, 6, 6]} bold>
```

Comment:
NIT: The line heights don't look proportional to me, kinda out of place. We can have a theme option in rendition to set line-heights based on font sizes.

Reply:
@iamsolankiamit I was hesitating whether I should remove it or leave it, so I figured we remove it first and change it only later if necessary, I'll let Alexandra be the judge here